pandas/_libs/tslib.pyx

```diff
@@ -67,6 +67,18 @@ from khash cimport (
     kh_init_int64, kh_int64_t,
     kh_resize_int64, kh_get_int64)
 
+from .tslibs import parsing  # noqa
+from .tslibs.parsing import ( # noqa
+    DateParseError,
+    _format_is_iso,
+    _DATEUTIL_LEXER_SPLIT,
+    _guess_datetime_format,
+    NAT_SENTINEL,
+    parse_datetime_string,
+    parse_time_string,
+    _does_string_look_like_datetime,
+    parse_datetime_string_with_reso)
```

Comment:
also those ones. Do you just import them to keep them in the namespace of tslib as before? Are they actually used from here somewhere?

Reply:
Most of these are just for backward compat in case some user somewhere is using them.  I'll be happy to remove those.

Reply:
same comment as above. don't import these in tslib.pyx unless they are actually used, rather change to directly import

Reply:
can u address these comments

Reply:
Sure, just pushed.  Retained the `from tslibs.parsing import DateParseError # noqa` as it is referenced in a handful of places around pandas (and conceivably downstream), removed all the others.